profile_library/roborock/s6_maxv/model.json

```diff
@@ -0,0 +1,120 @@
+{
+  "calculation_enabled_condition": "{{ is_state('[[entity]]', 'docked') }}",
+  "calculation_strategy": "linear",
+  "created_at": "2024-11-11T17:19:55.943226",
+  "device_type": "vacuum robot",
+  "linear_config": {
+    "attribute": null,
```

Comment:
Did the measure script not set this to `battery_level`?

Reply:
Looks like not. It's `battery_level` on this model and was auto-detected by the new feature I suggested. I guess that's a bug then.